deny.toml

```diff
@@ -56,4 +56,4 @@ allow-git = [
 
 [sources.allow-org]
 # github.com organizations to allow git sources for
-github = ["embassy-rs", "esp-rs", "kaspar030"]
+github = ["embassy-rs", "esp-rs", "kaspar030", "elenaf9"]
```

Comment:
reminder: drop this once embassy-rp commit is in `kaspar030/embassy`

Reply:
dropped.